client/locations-list-map.js

```diff
@@ -1163,6 +1215,37 @@ function zoomToMarker(marker) {
   }
 }
 
+// copies the direct URL for a given entry to clipboard
+function copyLinkToClipboard(rowId) {
+  const textArea = document.createElement('textarea');
+  Object.assign(textArea.style, {
+    position: 'fixed',
+    top: 0,
+    left: 0,
+    width: '2em',
+    height: '2em',
+    padding: 0,
+    border: 'none',
+    outline: 'none',
+    boxShadow: 'none',
+    background: 'transparent',
+  });
+
+  const url = new FtmUrl(window.location.href);
+  url.searchparams.id = rowId;
+
+  textArea.value = url.toString();
+  document.body.appendChild(textArea);
+  textArea.focus();
+  textArea.select();
+  try {
+    document.execCommand('copy');
```

Comment:
looks like this feature is deprecated: https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand

Do you think it's worth doing the async version with a fallback to execCommand? https://stackoverflow.com/questions/400212/how-do-i-copy-to-the-clipboard-in-javascript

Reply:
It does work great for me!

Reply:
Rats! I thought I was getting around having to do two things (with a fallback option) by just using the fallback option but if it's deprecated I guess I should try the new option first. Hahah